### PR TITLE
develop <- features/105_cleanup: cleanup the readme

### PR DESCRIPTION
**Pull request by blablanumerodeux**

### CODE REVIEW

File: .circleci/docker-compose-prod.yml [hunk 1]
```diff
@@ -1,39 +1,56 @@
-version: '3'
+version: '3.1'
```
Comment (shulolo) [inline .circleci/docker-compose-prod.yml:1]: Why this changes?

File: .circleci/docker-compose-prod.yml [hunk 2]
```diff
@@ -1,39 +1,56 @@
...
-  app-ng:
+
+  front:
     image: "dakarinternational/dakar-ng:latest"
-    ports:
-      - "80:80"
-    container_name: 'app-dakar-ng'
     depends_on:
-      - "app"
+      - "dakar-api"
```
Comment (shulolo) [inline .circleci/docker-compose-prod.yml]: if think that you just mean "api"?
But why the front is depending on the backend?

File: README.MD [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
-sudo systemctl start docker  
-sudo usermod -a -G docker $USER  
-sudo reboot  
-docker pull dreaserdocker/dakar
-docker-compose pull
-docker login
-```
+# Dakar
+
+Fully reactive online travel agency made with Spring Boot 2 (w/ WebFlux), Angular, GraphQL, CouchBASE, Maven, CircleCI, and Docker.
```
Comment (shulolo) [inline README.MD]: I think it's called couchbase not couchBASE

File: README.MD [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
-sudo reboot  
-docker pull dreaserdocker/dakar
-docker-compose pull
-docker login
-```
+# Dakar
+
+Fully reactive online travel agency made with Spring Boot 2 (w/ WebFlux), Angular, GraphQL, CouchBASE, Maven, CircleCI, and Docker.  
+
+The goal is to learn as much as possible about new tech while creating a small but hopefully helpful travel agency fully made in reactive programing.
```
Comment (shulolo) [inline README.MD]: you are better than me in english but I think it's "programming" with two "m"

File: README.MD [hunk 5]
```diff
@@ -1,42 +1,43 @@
...
+
+## Maven Central Repo
+https://mvnrepository.com/artifact/com.dakar/dakar
+
+## Docker hub 
+https://hub.docker.com/repository/docker/dakarinternational/dakar
 
+## Github Release
+https://github.com/DaKarInternational/DaKar/releases/tag/3.0.0
```
Comment (shulolo) [inline README.MD:14]: it seems that we should do a release for the 5.0.0, isn't it?

File: README.MD [hunk 6]
```diff
@@ -1,42 +1,43 @@
...
 
-#### On way to launch the app in local: 
+
+### On way to launch the app  
 1. build jar  
-`mvnw install -DskipTests  `
+`mvnw clean install  `
```
Comment (shulolo) [inline README.MD]: Will the jar be created if we do not skip the tests as the database is not launched?

File: README.MD [hunk 7]
```diff
@@ -1,42 +1,43 @@
...
 
 2. build docker  
-`mvnw dockerfile:build  `
+`./mvnw dockerfile:build  `
```
Comment (shulolo) [inline README.MD]: small remark, there is a space before you're close the quotes, at many locations in fact, I know I'm relou...

File: README.MD [hunk 8]
```diff
@@ -1,42 +1,43 @@
...
-add couch-dakar in your /etc/hosts -> this is temporary.  
 
 
-#### Launch all docker-compose services
-`docker-compose up  `  
+### Some test endpoints
 
-#### Some test endpoints
 Functional controller   
 http://localhost:8080/test5/
```
Comment (shulolo) [inline README.MD:42]: may be we should rename our endpoints....

File: src/main/resources/application-prod.properties [hunk 9]
```diff
@@ -5,7 +5,7 @@ spring.jpa.properties.hibernate.format_sql=true
 
 spring.data.couchbase.auto-index=true
 # address of the Couchbase Cluster nodes
-spring.couchbase.bootstrap-hosts=couch-dakar
+spring.couchbase.bootstrap-hosts=dakar_couch
```
Comment (shulolo) [inline src/main/resources/application-prod.properties:8]: I can't find it on docker compose file

**Review by shulolo — CHANGES_REQUESTED** [review]

File: .circleci/docker-compose-prod.yml [hunk 10]
```diff
@@ -1,39 +1,68 @@
...
-    ports:
-      - "8080:8080"
-    container_name: 'app-dakar'
     depends_on:
       - "couch"
     networks:
-      - dakar-network
+      #- dakar-network
+      - traefik_traefik
     environment:
```
Comment (shulolo) [inline .circleci/docker-compose-prod.yml:25]: how does it work?

File: .circleci/docker-compose-prod.yml [hunk 11]
```diff
@@ -1,39 +1,68 @@
...
+        - "traefik.http.routers.dakar-ssl-api.tls.certresolver=myresolver"
+
+  front:
     image: "dakarinternational/dakar-ng:latest"
-    ports:
-      - "80:80"
-    container_name: 'app-dakar-ng'
     depends_on:
-      - "app"
+      - "api"
```
Comment (shulolo) [inline .circleci/docker-compose-prod.yml]: why the front depends on the backend?

File: src/main/resources/application.properties [hunk 12]
```diff
@@ -1,2 +1,4 @@
 #spring.profiles.include=ssl
 spring.profiles.active=local,ssl
+
+server.port=8088
```
Comment (shulolo) [inline src/main/resources/application.properties:4]: why 8088?

**Review by shulolo — CHANGES_REQUESTED** [review]

File: .circleci/docker-compose-prod.yml (hunk 1 above)
Comment (blablanumerodeux) [reply .circleci/docker-compose-prod.yml:1]: just trying to be consistent with the other compose files

File: .circleci/docker-compose-prod.yml (hunk 2 above)
Comment (blablanumerodeux) [reply .circleci/docker-compose-prod.yml]: dependenci is non mandatory, you're right. it's fixed.

File: README.MD (hunk 7 above)
Comment (blablanumerodeux) [reply README.MD]: lol

File: src/main/resources/application-prod.properties (hunk 9 above)
Comment (blablanumerodeux) [reply src/main/resources/application-prod.properties:8]: on the compose file, you'll find the "couch" service. But the final hostname will be a combination of the swarm stack and the service name, means: dakar_couch

File: .circleci/docker-compose-prod.yml (hunk 10 above)
Comment (blablanumerodeux) [reply .circleci/docker-compose-prod.yml:25]: so, for now, all our services are on the same network. 
just the time for me to figure out how to manage multiple networks properly, without having Gateway Timeout exceptions...

File: src/main/resources/application.properties (hunk 12 above)
Comment (blablanumerodeux) [reply src/main/resources/application.properties:4]: cause the 8080 is already used by the dashboard of traefik.
also i could have put ANY unused port. it really doesn't mater as it's not exposed and as it's "reverse proxyed" by traefik...

**Review by shulolo — APPROVED** [review]
